docs/src/pages/guides/api/api.md

```diff
@@ -106,10 +106,10 @@ In the cases where there are more events, we put the noun first, and then the ve
 
 ### boolean vs. enum
 
-There are two options to design the API for the variations of a component: with a _boolean_; or with an _enum_.
+There are two options to design the API for the variations of a component: with a _Boolean_; or with an _enum_.
```

Comment:
Why is the built-in type upper case but the colloquial "enum" lower case? Note that there is a notion of `Boolean` in JS and we don't mean that here. We actually refer to the boolean primitive.

Reply:
I have changed it to be consistent, but lower case sounds good as well 👍

Reply:
Actually, maybe we should simply use the automatically generated API page as the source of truth for manually authored pages?

<img width="332" alt="Screenshot 2021-04-25 at 20 30 30" src="https://user-images.githubusercontent.com/3165635/116004920-2f16c200-a605-11eb-9392-124f08e96970.png">

https://next.material-ui.com/api/autocomplete/#props

Reply:
Alright, tracking in https://github.com/mui-org/material-ui/issues/25970